packages/rocketchat-ui/lib/chatMessages.coffee

```diff
@@ -39,9 +24,60 @@ class @ChatMessages
 			index++
 		return -1
 
+	recordInputAsDraft: () ->
+		console.log(this.editing)
```

Comment:
ye lol, sry


On 05/02/2016 08:08 PM, Rodrigo Nascimento wrote:
>
> In packages/rocketchat-ui/lib/chatMessages.coffee 
> <https://github.com/RocketChat/Rocket.Chat/pull/3102#discussion_r61768045>:
>
> > @@ -39,9 +24,60 @@ class @ChatMessages
> >  			index++
> >  		return -1
> >
> > +	recordInputAsDraft: () ->
> > +		console.log(this.editing)
>
> Can we remove this log?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/RocketChat/Rocket.Chat/pull/3102/files/a830464fedddaaa867d7e6d68dc0f77b38f1ae89#r61768045>
>